packages/local_auth/local_auth_darwin/darwin/local_auth_darwin.podspec

```diff
@@ -11,9 +11,9 @@ Downloaded by pub (not CocoaPods).
                        DESC
   s.homepage         = 'https://github.com/flutter/packages'
   s.license          = { :type => 'BSD', :file => '../LICENSE' }
-  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com', 'Om Phatak' => 'everythingoutdated@gmail.com' }
```

Comment:
This line should use the standard boilerplate; we don't list individual authors in this file.